pages/Documents/RichMessaging/AppleMessagesForBusinessTemplates/form-message-template.md

```diff
@@ -0,0 +1,348 @@
+
+  For all possible options for page configuration check [Apple Documentation](https://register.apple.com/resources/messages/msp-rest-api/type-interactive#form-message).
+
+#### Example Page Template
+
+```json
+{
+  "pageIdentifier":"1",
+  "type":"input",
+  "nextPageIdentifier":"2",
+  "title":"Your email address",
+  "subtitle":"Enter your email address",
+  "options":{
+    "required":true,
+    "inputType":"singleline",
+    "maximumCharacterCount":250,
+    "keyboardType":"emailAddress",
+    "textContentType":"emailAddress"
+  }
+}
+```
+
+### Example Body Template
+The structured content body will allow you to define the spalsh screen of the Business Form. 
+
+The Following Properties must be included in the Body template
```

Comment:
Please correct case (“following properties”, “body template”), add colon, and new line between paragraph and list.

Reply:
Done